### ISSUE TITLE
Stop working

### ISSUE THREAD

**Issue by Nimond** [zxqfl/tabnine-sublime#48]
TabNine stops work in my SublimeText 3.2.2 Build 3211
sublime-text console output:
```python
Traceback (most recent call last):
  File "/opt/sublime_text_3/sublime_plugin.py", line 610, in on_selection_modified_async
    callback.on_selection_modified_async(v)
  File "/home/dnomin/.config/sublime-text-3/Packages/TabNine/TabNine.py", line 334, in on_selection_modified_async
    response = tabnine_proc.request(request)
  File "/home/dnomin/.config/sublime-text-3/Packages/TabNine/TabNine.py", line 89, in request
    self.restart_tabnine_proc()
  File "/home/dnomin/.config/sublime-text-3/Packages/TabNine/TabNine.py", line 85, in restart_tabnine_proc
    startupinfo=get_startup_info(sublime.platform()))
  File "./python3.3/subprocess.py", line 819, in __init__
  File "./python3.3/subprocess.py", line 1370, in _execute_child
  File "./python3.3/os.py", line 791, in fsencode
TypeError: expect bytes or str, not NoneType
```

**Comment by TheCrashDown** [comment]
Seems like it caused by this commit - https://github.com/zxqfl/tabnine-sublime/commit/ea8244ef87930d6f52c07cc3fc42ad25ad43a7ca

Problem was fixed in last commit - https://github.com/zxqfl/tabnine-sublime/commit/5d8761d6ceeaccb2cc6e883008fdb6ad2bc517c3#diff-5c5733ecfc175b2cc067928438f28020. 

Renaming two files fixed problem for me

**Comment by PPetky** [comment]
On Decenmer the 13 - stopped working completely. Even the uninstall and cleanup does not helped. Something is very broken.

**Comment by Boscop** [comment]
I have the same issue!

**Comment by amircodota** [comment]
We released TabNine sublime plugin version 2.1.23 that fixes the issue.
Closing.

Feel free to re-open if you still encounter issues
